README.md

```diff
@@ -72,7 +72,8 @@ A curated list of amazingly awesome awesomeness.
 		- [by @Wolg](https://github.com/Wolg/awesome-swift)
 		- [from ZEEF by @Edubits](https://swift.zeef.com/robin.eggenkamp)
 	- [TypeScript](https://github.com/dzharii/awesome-typescript)
```

Comment:
Also, please remove the link to dzharii's list from the section title, and add it as another subsection, something like.

- Section title (no links)
 - by @oneperson
 - by @anotherperson